README.md

```diff
@@ -1,6 +1,9 @@
 # git-fukit
 redo a git repo a la xkcd.com/1597
```

Comment:
Would you mind making this change as well? I feel if I have the actual comic in the readme, I should have an actual link to it? What do you think?
```suggestion
redo a git repo a la [xkcd.com/1597](https://xkcd.com/1597/)
```